deliverable_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+read -p "Which CI/CD service do you use (Can be edited later) [(g)ithub/(b)itrise/(l)ater]: " ciService
+
+if [ "$ciService" = "g" -o "$ciService" = "github" ]; then
+    echo "Setting template for Github Actions"
+    rm bitrise.yml
+elif [ "$ciService" = "b" -o "$ciService" = "bitrise" ]; then
```

Comment:
I am a bit confused about the syntax inside the if [] . Are we checking if "$ciService" equal to "b" ? If we check equality then should we use == instead of =?